src/query/procedure/module.cpp

```diff
@@ -33,6 +34,32 @@ extern "C" {
 
 namespace memgraph::query::procedure {
 
+constexpr const char *func_code =
+    "import ast\n\n"
+    "no_removals = ['collections', 'abc', 'sys']\n"
+    "modules = set()\n\n"
+    "def visit_Import(node):\n"
+    "  for name in node.names:\n"
+    "    mod_name = name.name.split('.')[0]\n"
+    "    if mod_name not in no_removals:\n"
+    "      modules.add(mod_name)\n\n"
+    "def visit_ImportFrom(node):\n"
+    "  if node.module is not None and node.level == 0:\n"
+    "    mod_name = node.module.split('.')[0]\n"
+    "    if mod_name not in no_removals:\n"
+    "      modules.add(mod_name)\n"
+    "node_iter = ast.NodeVisitor()\n"
+    "node_iter.visit_Import = visit_Import\n"
+    "node_iter.visit_ImportFrom = visit_ImportFrom\n"
+    "node_iter.visit(ast.parse(code))\n";
+
+// NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)w
+DEFINE_string(python_submodules_directory, "mage",
```

Comment:
this might have a similar problem to the `-h` flag -> the `SHOW CONFIG` test has to be extended -> I'll merge the -h pr to master and update this pr because otherwise there might be a conflict 😄